test/db/tools/rabin2

```diff
@@ -1044,3 +1044,220 @@ EXPECT=<<EOF
 1
 EOF
 RUN
+
+NAME=rabin2 -g
+FILE=bins/elf/analysis/x86-helloworld-clang
```

Comment:
I can't find this file, where is it?

Reply:
I did a PR in the other project for this one.

Reply:
how does this test check the additional feature you have introduced? I have run the command on both master and your PR, but it gives the same output. Is there a more specific test we can use to check this is working well?

Reply:
Hi @devnexen . Is there anything we can do to move this forward?